tests/modelbench_tests/test_benchmark_runner.py

```diff
@@ -204,12 +241,15 @@ def test_benchmark_sut_worker_throws_exception(self, item_from_test, a_wrapped_t
         assert result.sut_response is None
         assert isinstance(result.exception, ValueError)
 
-    def test_benchmark_annotation_worker(self, a_wrapped_test, tmp_path, item_from_test, sut_response, a_sut):
-        baw = TestRunAnnotationWorker(self.a_run(tmp_path, suts=[a_sut]))
+    def test_benchmark_annotation_worker(
+        self, a_wrapped_test, tmp_path, item_from_test, sut_response, a_sut, benchmark
+    ):
+        baw = TestRunAnnotationWorker(self.a_run(tmp_path, suts=[a_sut], benchmarks=[benchmark]))
```

Comment:
I'm not following why this needs now need benchmarks.

Reply:
Because the worker gets the annotators from the tests in the benchmark. Without setting benchmarks here, it won't run any annotators and this test fails.

Reply:
That's a good point, I added another test for the TestRun/Runner. I kept the benchmark test because the BenchmarkRun and TestRun add tests differently and I think it's worth making sure it works for both,

Reply:
Sounds great!